CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - added: Add Solana init options
 - added: BSC staking on Thorchain Savers
 - added: Ability to clear datastore of fiat providers
+- added: 'numAccounts' parameter to report number of locally saved accounts
```

Comment:
`- added: 'numAccounts' tracking parameter to report number of locally saved accounts`